multiqc/utils/config.py

```diff
@@ -206,6 +206,27 @@ def load_sample_names(snames_file):
         logger.error("Error loading sample names file: {}".format(e))
     logger.debug("Found {} sample renaming patterns".format(len(sample_names_rename_buttons)))
 
+def load_show_hide(sh_file):
+    global show_hide_buttons, show_hide_patterns, show_or_hide
+    try:
+        with open(sh_file) as f:
+            logger.debug("Loading sample renaming config settings from: {}".format(sh_file))
+            show_hide_buttons.append("all")
+            show_hide_patterns.append("")
+            show_or_hide.append("show")
+            for l in f:
+                s = l.strip().split("\t")
+                if len(s) == 3:
+                    show_hide_buttons.append(s[0])
+                    show_hide_patterns.append(s[1])
+                    if not s[2] in ["show", "hide"]:
+                        raise IOError
```

Comment:
Maybe an `assert` statement would be better here..

Reply:
Instead of this I just added to the if statement, so that it skips any lines that don't meet our expectations instead of stopping the parsing of the whole file